documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -80,7 +80,7 @@ Perform a site-level SIPNET run using the following settings
   </spin.up>
   <forecast.time.step>1</forecast.time.step>
   <start.date>1980/01/01</start.date>
-  <end.date>2014/12/31</end.date>
+  <end.date>2016/12/31</end.date>
```

Comment:
I think `git pull git@github.com:PalEON-Project/Camp2016.git master` should use the https address : https://github.com/PalEON-Project/Camp2016.git

also this [line](https://github.com/PecanProject/pecan/pull/992/files#diff-d08854c9e1900ba11320aa6d3e2abb8aR89) could be: 

> In the PEcAn History, go to your PDA run and open ```pecan.pda[UNIQUEID].xml``` (the one pecan saves for your AFTER you finished the PDA)